qendpoint-core/src/main/java/com/the_qa_company/qendpoint/core/storage/iterator/CloseableAttachQueryIterator.java

```diff
@@ -12,7 +12,7 @@ public class CloseableAttachQueryIterator implements QueryCloseableIterator {
 
 	@SafeVarargs
 	public static QueryCloseableIterator of(QueryCloseableIterator it,
-			AutoCloseableGeneric<QEPCoreException>... closeables) {
+			AutoCloseableGeneric<? extends RuntimeException>... closeables) {
 		Objects.requireNonNull(it, "it can't be null!");
 		if (closeables.length == 0) {
 			return it;
```

Comment:
Again I'm not sure if this is the correct approach to fixing the issues when migrating to RDF4J 5.0.0.